File: src/sql_jsc/postgres/types/bool.zig
pub const to = 16;
pub const from = [_]short{16};

pub fn toJS(
    _: *jsc.JSGlobalObject,
    value: bool,
) AnyPostgresError!JSValue {
    return JSValue.jsBoolean(value);
}

const bun = @import("bun");
const AnyPostgresError = @import("../../../sql/postgres/AnyPostgresError.rust").AnyPostgresError;

const int_types = @import("../../../sql/postgres/types/int_types.rust");
const short = int_types.short;

const jsc = bun.jsc;
const JSValue = jsc.JSValue;

File: src/sql_jsc/postgres/types/bool.rs
use crate::jsc::{JSGlobalObject, JSValue};
use bun_sql::postgres::AnyPostgresError;
use bun_sql::postgres::types::int_types::short;

pub const TO: short = 16;
pub const FROM: [short; 1] = [16];

pub fn to_js(_: &JSGlobalObject, value: bool) -> Result<JSValue, AnyPostgresError> {
    Ok(JSValue::js_boolean(value))
}

// ported from: src/sql_jsc/postgres/types/bool.zig
